src/components/video-card/VideoCard.jsx

```diff
@@ -1,9 +1,15 @@
 import "./video-card.css";
 import { MdOutlineWatchLater, MdPlaylistAdd } from "react-icons/md";
-import { AiOutlineLike } from "react-icons/ai";
+import { AiOutlineLike, AiFillLike } from "react-icons/ai";
+import { useLike } from "../../context/like-context";
+import { isPresentInData } from "../../utils/isPresentInData";
+import { useAuth } from "../../context/auth-context";
```

Comment:
you can import all the context from 1 import statement using index.js

Reply:
Done.